code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -109,7 +109,7 @@
 
 /obj/item/clothing/gloves/thick/swat/blackshield_corpsman
 	name = "corpsman combat gloves"
-	desc = "A pair of fire and impact-resistant corpsman combat gloves. This pair has an extra rubbery coating to help prevent germs."
+	desc = "A pair of fire and impact-resistant corpsman combat gloves." // No lying, it's not anti-germs.
```

Comment:
it was to show it has extra bio armor

Reply:
But it doesn't prevent germs and you didn't let me assign it the var Nitriles have >:|